Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/CopyPass.h

```diff
@@ -80,15 +79,25 @@ namespace AZ
                 Invalid
             };
             CopyMode m_copyMode = CopyMode::Invalid;
+            bool m_inputOutputCopy = false;
 
             constexpr static int MaxFrames = RHI::Limits::Device::FrameCountMax;
+
+            struct PerAspectCopyInfo
+            {
+                RHI::CopyItem m_copyItemDeviceToHost{};
+                RHI::CopyItem m_copyItemHostToDevice{};
+                AZStd::array<Data::Instance<Buffer>, MaxFrames> m_device1HostBuffer{};
+                AZStd::array<Data::Instance<Buffer>, MaxFrames> m_device2HostBuffer{};
+                AZStd::array<AZ::u64, MaxFrames> m_deviceHostBufferByteCount{};
+                RHI::DeviceImageSubresourceLayout m_inputImageLayout;
+            };
+
+            AZStd::vector<PerAspectCopyInfo> m_perAspectCopyInfos;
```

Comment:
Minor: Lets add a comment explaining the purpose of this vector and why it is setup as one.